test/test_browser.py

```diff
@@ -4783,7 +4783,7 @@ def test_webgl_simple_enable_extensions(self):
 
   @parameterized({
     '': ([],),
-    'closure': (['-sASSERTIONS', '--closure=1'],),
+    'closure': (['-sASSERTIONS', '--closure=1', '-sEXPORTED_FUNCTIONS=_malloc,_free'],),
```

Comment:
Add `malloc` and `free` to the `__deps` so this won't be necessary. (Otherwise everyone who uses GetCompilationInfo will need to add these linker options.) Example:
https://github.com/emscripten-core/emscripten/blob/28a09f695973f99d784cc344fcac6289452a2240/src/library_wget.js#L63C64-L63C64